kaolin/graphics/SoftRenderer.py

```diff
@@ -180,7 +196,7 @@ def __init__(
             # TODO: Add comments here
             self.viewing_angle = viewing_angle
             # TODO: use kal.deg2rad instead
-            self.eye = torch.FloatTensor([0, 0, -(1. / torch.tan(kal.math.pi
+            self.eye = torch.FloatTensor([0, 0, -(1. / torch.tan(kal.mathutils.pi
```

Comment:
Please don't use `kaolin` module inside kaolin core.

Reply:
Can you suggest a best practice here? I'll make changes accordingly

Reply:
Use relative imports within Kaolin core. Also, Why not `math.radians(self.viewing_angle)` since it’s a float?